open-apparel-registry/open-apparel-registry
# Display claim information

## Overview

Ports the facility claim information section from the old sidebar to
the new sidebar, removes old sidebar components, and adds a 
claimed badge to claim items. 

Connects #1765 

## Demo

<img width="356" alt="Screen Shot 2022-06-14 at 9 18 57 AM" src="https://user-images.githubusercontent.com/21046714/173590560-6de479e7-def0-4249-9aaa-61d972d6719f.png">
<img width="358" alt="Screen Shot 2022-06-14 at 9 19 08 AM" src="https://user-images.githubusercontent.com/21046714/173590576-1b2a7f21-f7cf-4b5b-bce2-775bbd9699db.png">
<img width="354" alt="Screen Shot 2022-06-14 at 9 19 29 AM" src="https://user-images.githubusercontent.com/21046714/173590600-1d56ef71-5221-450a-b34f-9ce54be0e8f0.png">
<img width="352" alt="Screen Shot 2022-06-14 at 9 19 52 AM" src="https://user-images.githubusercontent.com/21046714/173590616-96b47f12-19e4-49e7-bdfb-b87673194302.png">

## Testing Instructions

* Run `./scripts/server`
* Visit an unclaimed facility and confirm that the details load as before. 
* Claim the facility. Approve the claim as an admin user. 
* Visit the claimed facility and view the additional 'description' field. Scroll over the badge to see the tooltip.
* Fill out the full claim form and make all details public. View the claimed facility and confirm the name, address, and other claim fields are integrated and show badges. 
* Make some details private in the claim form. View the claimed facility and confirm those fields are hidden.
* The claim feature is disabled in embed mode. View the claimed facility in an embedded map. Confirm that the claim badges and claim fields are hidden.

## Checklist
